.github/workflows/publish.yaml

```diff
@@ -41,3 +40,18 @@ jobs:
           tags: |
             ghcr.io/rode/ui:latest
             ghcr.io/rode/ui:${{ steps.gitextra.outputs.version }}
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create Release
+        uses: actions/create-release@v1
```

Comment:
Any concerns about this action being unmaintained? https://github.com/actions/create-release#github-action---releases-api

Reply:
Good call out. I'll try to find another one.